seed/building_sync/tests/test_building_sync.py

```diff
@@ -54,10 +71,48 @@ def test_export(self):
         )
 
         ps = self.property_state_factory.get_property_state(organization=self.org)
-        print(obj_to_dict(ps))
+        ps.extra_data['floors_below_grade'] = 11235
+        ps.save()
+        # print(obj_to_dict(ps))
 
         xml = self.bs.export(ps, BuildingSync.BRICR_STRUCT)
-        print(xml)
+
+        self.assertTrue("<auc:FloorsBelowGrade>11235</auc:FloorsBelowGrade>" in xml)
+        self.assertTrue("<auc:State>Oregon</auc:State>" in xml)
+
+        # check for complicated fields and measures.
+
+    def test_set_node(self):
+        data = {
+            "a": 1,
+            "c": {
+                "d": 2
+            },
+            "d": {
+                "e": {
+                    "f": {
+                        "g": {
+                            "h": "deep"
+                        }
+                    }
+                }
+            }
+        }
+
+        result = self.bs._set_node('', data, 1999)
+        self.assertFalse(result)
+
+        result = self.bs._set_node('a', data, 1999)
+        self.assertTrue(result)
+        self.assertEqual(data["a"], 1999)
+
+        result = self.bs._set_node('c.d', data, "string_me")
+        self.assertTrue(result)
+        self.assertEqual(data["c"]["d"], "string_me")
+
+        # result = self.bs._set_node('d.e.f.g.h', data, 1.234)
+        # self.assertTrue(result)
+        # self.assertEqual(data["d"]["e"][""], )
```

Comment:
Any reason this test is commented?  It seems like it should work...

Reply:
no. it shouldn't be there. working on this a bit more today.